libmscore/stafftypechange.cpp

```diff
@@ -144,6 +144,13 @@ QVariant StaffTypeChange::getProperty(Pid propertyId) const
                   return _staffType->userMag();
             case Pid::SMALL:
                   return _staffType->small();
+            case Pid::STAFF_INVISIBLE:
+                  return _staffType->invisible();
+            case Pid::COLOR:
+                  if (_staffType)
+                        return _staffType->color();  //Exception if _staffType null. (Check why - this shouldn't be reached)
+                  else
+                      return QColor(Qt::red);  //black.
```

Comment:
This 'if (_staffType)' is necessary to avoid an exception, because the generic Pid::COLOR is being used for staffType._color. 
While getting basic properties from staffTypeChange as an element, MuseScore tries to read the element's COLOR, and the stc tries to return _staffType->color() before _staffType has been set.
Maybe it would be safer to create a STAFF_COLOR property id.